tools/cache_perf_tool/processEntry.go

```diff
@@ -74,6 +85,16 @@ func (p *processEntryGen) nodeWrites(ctx context.Context, node *sgwNode, delay t
 	ticker := time.NewTicker(delay)
 	defer ticker.Stop()
 	for {
```

Comment:
Should this be a function to deduplicate above where this passes in a time.Ticker with 0 so that the first one is deduplicated from this?

If the delay is actually 0, are we supposed to run this function twice?

Reply:
Can't pass 0 into ticket hence why I just went for this approach. No wouldn't expect this to run twice if node has 0 delay.